packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1975,6 +2001,9 @@ class EditableText extends StatefulWidget {
   /// {@macro flutter.widgets.magnifier.intro}
   final TextMagnifierConfiguration magnifierConfiguration;
 
+  /// The default value for [stylusHandwritingEnabled].
+  static const bool defaultStylusHandwritingEnabled = true;
```

Comment:
Why is the default exposed?

Reply:
In order to reference it from other files that also need the same default, instead of duplicating it.